README.md

```diff
@@ -46,34 +47,158 @@ Table of Contents
          * [Return an empty collection when there are no results](#return-an-empty-collection-when-there-are-no-results)
       * [Use null or unset keys that are not set](#use-null-or-unset-keys-that-are-not-set)
       * [Status Codes](#status-codes)
-         * [TODO](#todo-1)
+         * [TODO](#todo)
    * [Auth](#auth)
-      * [TODO](#todo-2)
+      * [TODO](#todo-1)
    * [Error Handling](#error-handling)
-      * [TODO](#todo-3)
+      * [TODO](#todo-2)
    * [Localization](#localization)
-      * [TODO](#todo-4)
+      * [TODO](#todo-3)
    * [Timeouts](#timeouts)
-      * [TODO](#todo-5)
+      * [TODO](#todo-4)
    * [Pagination](#pagination)
-      * [TODO](#todo-6)
+      * [TODO](#todo-5)
    * [Inspiration](#inspiration)
    
 # Requests
 
 ## URLs
 
-### TODO
+### Prefix API endpoints
+
+Prefix API endpoints with `/api/` to separate them from other URLs like HTML views served on the same server.
 
 <details>
 <summary>Click to see examples</summary>
 
 #### ✅
 
-#### ⛔️
+Make sure to prefix your API endpoints with `/api/`:
+
+```bash
+www.example.com/api/v1/products/1
+```
 
 </details>
 
+### API versioning
+
+Versioning your API allows you to make non-backwards compatible changes to your API for newer clients by introducing new versions of endpoints while not breaking existing clients.
+
+Include the API version in the URL. Versions start with 1 and are prefixed with `v`. The version path component should come right after the [`api`](#prefix-api-endpoints) path component.
+
+In case of an existing API that doesn't have this versioning scheme but needs a new version, skip `v1` and go straight to `v2`.
+
+> We're not recommending to use the headers (typically the Accept header) for versioning. The URL based approach is more obvious, usually simpler to implement, and testing URLs can be done in the browser.
+
```

Comment:
Should the example below also be foldable, or was it left visible on purpose?

Reply:
fixed :)